suites/pacific/upgrades/tier-2_upgrade_test-multi-path-upgrade-to-5.1-latest.yaml

```diff
@@ -25,53 +27,58 @@ tests:
             ansi_config:
               osd_scenario: lvm
               osd_auto_discovery: False
-              copy_admin_key: true
+              copy_admin_key: True
               dashboard_enabled: True
               dashboard_admin_user: admin
               dashboard_admin_password: p@ssw0rd
               grafana_admin_user: admin
               grafana_admin_password: p@ssw0rd
               cephfs_pools:
                 - name: "cephfs_data"
-                  pgs: "8"
+                  pgs: "32"
                 - name: "cephfs_metadata"
-                  pgs: "8"
-        5.0:
+                  pgs: "32"
+        4.3:
           config:
             ansi_config:
               osd_scenario: lvm
               osd_auto_discovery: False
-              copy_admin_key: true
+              copy_admin_key: True
               dashboard_enabled: True
               dashboard_admin_user: admin
               dashboard_admin_password: p@ssw0rd
               grafana_admin_user: admin
               grafana_admin_password: p@ssw0rd
-              containerized_deployment: true
-              upgrade_ceph_packages: True
               cephfs_pools:
                 - name: "cephfs_data"
-                  pgs: "8"
+                  pgs: "32"
```

Comment:
check if this can be set to `auto tune`?

Reply:
Hi Sunil,
As discussed I will try out the auto tune changes and pick it up in a new PR since this is not the core functionality of this PR.